streamer/output_stream.py

```diff
@@ -36,6 +36,7 @@ def __init__(self,
     self.input: Input = input
     self.features: Dict[str, str] = {}
     self.codec: Union[AudioCodec, VideoCodec, None] = codec
+    self.forced_subtitle = False;
```

Comment:
isn't that already in `Input`? why repeat it again here if we could use `self.input.forced_subtitles` when we need to query the value

Reply:
I have simplified the code, and I have also followed your suggestion since this only applies to subtitles.

Reply:
well, i mean that `self.input` declared 3 lines above already has this data, so we don't need to copy it further in output stream. so `forced_subtitle` field here isn't really that useful.

Reply:
Done!